code/controllers/subsystems/trade.dm

```diff
@@ -38,14 +39,14 @@ SUBSYSTEM_DEF(trade)
 	var/list/stations2init = collect_spawn_always()
 
 	while(trade_stations_budget && length(weightstationlist))
-		var/datum/trade_station/s = pickweight(weightstationlist)
-		if(istype(s))
-			stations2init += s
-		s.cost_trade_stations_budget()
-		weightstationlist.Remove(s)
+		var/datum/trade_station/station_instance = pickweight(weightstationlist)
+		if(istype(station_instance))
+			stations2init += station_instance
+			station_instance.cost_trade_stations_budget()
+		weightstationlist.Remove(station_instance)
 	init_stations_by_list(stations2init)
 
-/datum/controller/subsystem/trade/proc
+/datum/controller/subsystem/trade/proc/
 	collect_trade_stations()
```

Comment:
This is incredibly heretical

Reply:
![image](https://user-images.githubusercontent.com/46600554/101669446-836d3b00-3a84-11eb-8c64-9d63ef009f92.png)